config/locales/activerecord.models.pt.yml

```diff
@@ -0,0 +1,20 @@
+# encoding: UTF-8
+
+pt:
+  activerecord:
+    models:
+      adm:
+        one:   "Administrador"
+        other: "Administradores"
```

Comment:
one and other? you sure this is the best name?

Reply:
don't forget to mention the i18n documentation.

Reply:
I tried to use "_plural" as the [i18n documentation says](https://www.i18next.com/translation-function/plurals) but it didn't work, so I used "other" as shown in the [rails documentation](https://guides.rubyonrails.org/i18n.html#pluralization).

Apparently "plural" does not work because the gem did not map this pluralization by default, so it would have to be configured manually.